doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+SmartSim entity, use the ``model.colocate_db()`` factory method.
+In this case, the database
+is created via the SmartSim Model API function ``model.colocate_db``.
+The :ref:`Model API<model_api>` is accessed once a ``Model`` object has been initialized.
+
+
+Multi-db support
+^^^^^^^^^^^^^^^^
+SmartSim supports multi-database functionality, enabling an experiment
+to have several concurrently launched ``Orchestrator(s)``. If there is
+a need to launch more than one ``Orchestrator``, the ``Experiment.create_database()``
+function mandates the specification of a unique database identifier,
+denoted by the `db_identifier` argument, per created orchestrator.
+
+The `db-identifier` is used to reference SmartSim
+orchestrator(s) from application client code. This is particularly
+useful in instances where an orchestrator is colocated with a SmartSim
+model for low-latency inference and another Orchestrator is launched to
+handle other aspects of the workflow such as visualization and ML model
+training. More detailed information on the ideal use cases for orchestrator(s)
+and co-located ``Orchestrator(s)`` are available in sections... (update this when use cases added)
 
-In addition to a single model, SmartSim has the ability to launch an
-``Ensemble`` of ``Model`` applications simultaneously.
-
-An ``Ensemble`` can be constructed in three ways:
-  1. Parameter expansion (by specifying ``params`` and ``perm_strat`` argument)
-  2. Replica creation (by specifying ``replicas`` argument)
-  3. Manually (by adding created ``Model`` objects) if launching as a batch job
+Model
+-----
+``Model(s)`` represent any computational kernel, including applications,
+scripts, or generally, a program.
+They can interact with other
+SmartSim entities via data transmitted to/from SmartSim Orchestrators
+using a SmartRedis client.
+Models in PT, TF, and ONNX (scikit-learn, spark, and others) can be
+written in Python and called from Fortran or any other client languages.
+The Python code executes in a C runtime without the Python interpreter.
+
+Create a Model
+^^^^^^^^^^^^^^
+A ``Model`` is created through the function: ``Experiment.create_model()``.
+For initialization, models require ``RunSettings`` objects that specify
+how a kernel should be executed with regard to the workload manager
+(e.g., Slurm) and the available compute resources on the system.
+Optionally, the user may also specify a ``BatchSettings`` object if
+the model should be launched as a batch job on the WLM system.
+The ``create_model()`` factory method returns an initialized ``Model`` object that
+gives you access to functions associated with the :ref:`Model API<mode_api>`.
 
+Ensemble
+--------
+In addition to a single model, SmartSim offers the ability to run an
+``Ensemble`` of ``Model`` applications, i.e. multiple replicas of the simulation.
+More specifically, you can create, configure and launch groups of workloads (Ensembles)
```

Comment:
This is a bit of a redundant sentence, suggest combining with the above, i.e. "SmartSim allows users to create, configure, and launch an ``Ensemble`` of ``Model`` objects